examples/crm/src/misc/getContactAvatar.spec.ts

```diff
@@ -0,0 +1,54 @@
+/**
+ * @jest-environment node
+ */
```

Comment:
Why do you need that?

Reply:
For hashing the email for Gravatar, we use TextEncoder. 
However, it seems that this API is not embedded with the version of jsdom used by jest on your pipeline. If you have a solution, I'm interested